fastlane/Fastfile

```diff
@@ -99,16 +99,11 @@ platform :ios do
 
     clean_build_artifacts
     commit_version_bump(xcodeproj: "godtools.xcodeproj",
-                        message: "Build number bump for Beta build #{lane_context[SharedValues::BUILD_NUMBER]}",
+                        message: "Build number bump for Beta build #{lane_context[SharedValues::BUILD_NUMBER]} [ci skip]",
                         force: true)
     add_git_tag
-    push_to_git_remote(local_branch: ENV['GIT_BUILD_NUMBER_BRANCH'] || git_branch,
+    push_to_git_remote(local_branch: ENV['GIT_BRANCH'] || git_branch,
```

Comment:
If you are going to change this you will also need to change the cru_git_setup helper method that is run at the beginning of the beta and production lanes.

You should also make all these same changes to the production lane.

Once you have a working solution make these changes to the Fastfile.template so that new projects get these fixes too. It is found here https://github.com/CruGlobal/mobile-deployment-config/blob/master/ios/setup/Fastfile.template